### PR TITLE
move ec2 files to ec2/minimal

### PR DESCRIPTION
**Pull request by johnkim76**
- Moved ec2 files to ec2 minimal in preparation to create the ec2/multi_node
- made changes to env files and scripts for the move

### CODE REVIEW

File: ec2/multi_node/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Multi-Node AWS OpenShift Deployment 
+OpenShift environment with the Service Catalog & Ansible Service Broker configured in AWS with multiple EC-2 instances
+
+TBD
```
Comment (djwhatle) [inline ec2/multi_node/README.md]: What is this TBD here for?

File: ec2/minimal/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# CATalogASB EC-2 Deployment
```
Comment (djwhatle) [inline ec2/minimal/README.md]: Can we change this line to match the convention used above? CatASB instead of CATalogASB?

Comment (johnkim76) [reply ec2/minimal/README.md]: yes... will update

File: ec2/multi_node/README.md (hunk 1 above)
Comment (johnkim76) [reply ec2/multi_node/README.md]: I can remove it.. the contents will be updated once the files for the ec2 multi configuration gets placed in there

**Comment by djwhatle** [comment]
ACK, was able to deploy successfully with this.

**Pull request merged**